CP1404_practicals/prac_05/emails.py

```diff
@@ -1,25 +1,31 @@
 """Users email information"""
```

Comment:
It's a good idea to add a module docstring to explain what your code is doing. Week 3 Functions shows how to do  this.